### PR TITLE
Normalize import naming and replace goconf to go-toml

### PR DESCRIPTION
**Pull request by orsonwang**

### CODE REVIEW

File: TODO.md [hunk 1]
```diff
@@ -1 +1,2 @@
-See [issues](https://github.com/uniqush/uniqush-push/issues)
+Use NSQ to send push messages
```
Comment (monnand) [inline TODO.md]: Any reason why we need to change TODO? Right now, we are tracking our progress in the issues.

Why we want to NSQ?

File: TODO.md [hunk 2]
```diff
@@ -1 +1,2 @@
...
+Use SQLite, MySQL or MSSQL as database backend
```
Comment (monnand) [inline TODO.md]: For one, there's already an issue that I'm considering to use a relational database to store our data. But that is considered as a plan for version 2.0. The current database interface is not suitable for relational database so far as I can tell.

File: configparser.go [hunk 3]
```diff
@@ -18,13 +18,14 @@
...
 import (
-	"github.com/ifwe/goconf/conf"
-	. "github.com/uniqush/log"
-	. "github.com/uniqush/uniqush-push/db"
-	. "github.com/uniqush/uniqush-push/push"
 	"io"
 	"os"
 	"strings"
+
+	conf "github.com/pelletier/go-toml"
```
Comment (monnand) [inline configparser.go]: toml is nice. But we do not want to break our API, not until we bump to version 2.0

File: db/dbconfig.go [hunk 4]
```diff
@@ -19,7 +19,7 @@ package db
 
 import (
 	"fmt"
-	. "github.com/uniqush/uniqush-push/push"
+	"github.com/orsonwang/uniqush-push/push"
```
Comment (monnand) [inline db/dbconfig.go]: github.com/orsonwang is not a package that under our control.

**Comment by monnand** [comment]
Sorry, I don't know if this PR is sent by accident or it's intentional. I took a brief look at it and I'm afraid that it cannot be merged.

Despite some specific points I mentioned in the inline comments, some general rules about send a PR:

- Make your PR short, small. It should do *one* thing and one thing *only*.
- In case you want to contribute to this repo, set up your environment like this:
  - create a path ``github.com/uniqush`` under your ``$GOPATH``
  - fork this repo under your account
  - git clone *your* fork under directory ``$GOPATH/github.com/uniqush``
  - Once you've done that, you don't need to change the import path.
- You can have as many commits as you want while developing on your branch. But if you want to send a PR, squash them.

File: TODO.md (hunk 1 above)
Comment (orsonwang) [inline TODO.md]: Sorry. I accentually put my own plan to TODO.md.

**Pull request closed without merging**